tugboat/parser_engine/config/excel_spec.yaml

```diff
@@ -0,0 +1,51 @@
+---
+specs:
+    # Source MTN_57_AEC_Design_Specs_v_1.1.xlsx
+    spec1:
```

Comment:
Documentation is required on how to chose spec and what the user have to update in the spec. Check if you can avoid some trivial things like end_row by some means of logic in code.